index.html

```diff
@@ -3,9 +3,25 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="main.css">
   <script src="main.js" defer></script>
-  <title>Music Search</title>
+  <title>Matt's Music</title>
 </head>
 <body>
+  <form id = "music-search" style="display: grid; margin:10em;">
+    <input type="search" placeholder="Name of Artist" id="music-input" class="input-field"></input>
+    <div id="search-terms">
```

Comment:
Consider putting some validation in place for this input to prevent sending off a search request without a search term.